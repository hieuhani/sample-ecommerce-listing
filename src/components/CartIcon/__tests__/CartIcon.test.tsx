import React from 'react'
import renderer from 'react-test-renderer'
import { CartIcon } from '..'

describe('CartIcon', () => {
  test('CartIcon renders correctly', () => {
    const tree = renderer.create(<CartIcon />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
