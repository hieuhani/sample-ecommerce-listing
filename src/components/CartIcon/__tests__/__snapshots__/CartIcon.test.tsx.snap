// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartIcon CartIcon renders correctly 1`] = `
.c0 {
  width: 24pt;
  height: 24pt;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border: none;
  background-color: transparent;
  position: relative;
}

.c1 {
  background-color: #f58400;
  color: #FFF;
  min-width: 13pt;
  height: 13pt;
  border-radius: 50%;
  position: absolute;
  top: -5pt;
  right: -5pt;
  font-size: 10pt;
}

<button
  className="c0"
>
  <img
    src="/static/images/CartOutlined.png"
    srcSet="/static/images/CartOutlined@2x.png 2x, /static/images/CartOutlined@3x.png 3x"
  />
  <span
    className="c1"
  >
    1
  </span>
</button>
`;
